sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusSessionReceiverOptions.cs

```diff
@@ -0,0 +1,92 @@
+        /// whether the receiver is actively receiving, intended to help maximize throughput by allowing the receiver to receive
+        /// from a local cache rather than waiting on a service request.
+        /// </summary>
+        public int PrefetchCount
+        {
+            get
+            {
+                return _prefetchCount;
+            }
+            set
+            {
+                Argument.AssertAtLeast(value, 0, nameof(PrefetchCount));
+                _prefetchCount = value;
+            }
+        }
+        private int _prefetchCount = 0;
+
+        /// <summary>
+        /// The <see cref="ReceiveMode"/> used to specify how messages are received. Defaults to PeekLock mode.
+        /// </summary>
+        public ReceiveMode ReceiveMode { get; set; } = ReceiveMode.PeekLock;
+
+        /// <summary>
+        /// An optional session ID to scope the <see cref="ServiceBusSessionReceiver"/> to. If left blank,
+        /// the next available session returned from the service will be used.
+        /// </summary>
+        public string SessionId { get; set; }
+
+        /// <summary>
+        /// Determines whether the specified <see cref="System.Object" /> is equal to this instance.
+        /// </summary>
+        ///
+        /// <param name="obj">The <see cref="System.Object" /> to compare with this instance.</param>
+        ///
+        /// <returns><c>true</c> if the specified <see cref="System.Object" /> is equal to this instance; otherwise, <c>false</c>.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override bool Equals(object obj) => base.Equals(obj);
+
+        /// <summary>
+        /// Returns a hash code for this instance.
+        /// </summary>
+        ///
+        /// <returns>A hash code for this instance, suitable for use in hashing algorithms and data structures like a hash table.</returns>
+        ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override int GetHashCode() => base.GetHashCode();
+
+        /// <summary>
+        /// Converts the instance to string representation.
+        /// </summary>
+        ///
+        /// <returns>A <see cref="System.String" /> that represents this instance.</returns>
+        ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override string ToString() => base.ToString();
+
+        /// <summary>
+        /// Creates a new copy of the current <see cref="ServiceBusReceiverOptions" />, cloning its attributes into a new instance.
+        /// </summary>
+        ///
+        /// <returns>A new copy of <see cref="ServiceBusReceiverOptions" />.</returns>
+        internal ServiceBusSessionReceiverOptions Clone() =>
+            new ServiceBusSessionReceiverOptions
+            {
+                ReceiveMode = ReceiveMode,
+                PrefetchCount = PrefetchCount
```

Comment:
sessionId is missing here.

Reply:
Good catch - deleted method since it isn't actually used.